src/inc/fixuppointer.h

```diff
@@ -64,6 +64,20 @@ class RelativePointer
         return dac_cast<PTR_TYPE>(base + m_delta);
     }
 
+    // Returns value of the embedded encoded pointer with DPTR struct base. Assumes that the pointer is not NULL.
+    template<typename TYPE_T>
+    PTR_TYPE GetValueEmbedded(DPTR(TYPE_T) pStructBase) const
+    {
+        TYPE_T *pHostStructBase = (TYPE_T *) pStructBase;
```

Comment:
Casting from `DPTR(T)` to `T*` in DAC does not work. I do not think that this is correct.